customer-config/checkout-app/cha.settings.v1.json

```diff
@@ -326,6 +331,30 @@
         }
       }
     },
+    "MposPromptType": {
+      "type": "object",
+      "description": "Mpos prompt settings",
+      "properties": {
+        "scanner": {
+          "type": "object",
+          "description": "Mpos prompt camera scanner settings",
+          "properties": {
+            "disabledPromptCodes": {
+              "description": "List of prompt codes where camera scanner should not be showed",
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "default": [
+                "quantity.enter.quantity.prompt.quantitykey.message",
+                "markdown.order.entry.percentage.message"
+              ],
```

Comment:
Make sure to apply the defaults in the common repository 
https://github.com/extenda/experience-checkout-app-common/blob/master/customer-config/ccc.yaml